### PR TITLE
Add automount Flag to VolumeCreate Request

### PR DESCRIPTION
**Pull request by LKaemmerling**

### CODE REVIEW

File: hcloud/volume.go [hunk 1]
```diff
@@ -156,6 +158,9 @@ func (o VolumeCreateOpts) Validate() error {
 	if o.Server != nil && o.Location != nil {
 		return errors.New("only one of server or location must be provided")
 	}
+	if o.Server == nil && (o.Automount != nil && *o.Automount == true) {
```
Comment (thcyron) [inline hcloud/volume.go]: Drop the `== true`.

File: hcloud/volume.go [hunk 2]
```diff
@@ -188,6 +193,13 @@ func (c *VolumeClient) Create(ctx context.Context, opts VolumeCreateOpts) (Volum
 			reqBody.Location = opts.Location.Name
 		}
 	}
+	if opts.Automount != nil {
```
Comment (thcyron) [inline hcloud/volume.go]: Condition not needed. You can set `Automount` directly when constructing `reqBody`.

File: hcloud/volume.go [hunk 3]
```diff
@@ -188,6 +193,13 @@ func (c *VolumeClient) Create(ctx context.Context, opts VolumeCreateOpts) (Volum
...
+		reqBody.Automount = opts.Automount
+	}
+	if opts.Format != nil {
```
Comment (thcyron) [inline hcloud/volume.go]: Same here.

File: hcloud/server_test.go [hunk 4]
```diff
@@ -342,6 +342,59 @@ func TestServersCreateWithoutSSHKeys(t *testing.T) {
...
+	ctx := context.Background()
+	result, _, err := env.Client.Server.Create(ctx, ServerCreateOpts{
+		Name:       "test",
+		ServerType: &ServerType{ID: 1},
+		Image:      &Image{ID: 2},
+		Volumes: []*Volume{
+			{ID: 1},
+			{ID: 2},
+		},
+		Automount: nil, //Bool(true),
```
Comment (LKaemmerling) [inline hcloud/server_test.go]:
```suggestion
		Automount: Bool(true),
```

**Pull request merged**